Gemfile

```diff
@@ -67,6 +66,13 @@ group :development do
   #gem 'rack-mini-profiler'
 end
 
+group :test do
+  gem 'minitest-reporters'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.